### DISCUSSION TITLE
Putasset(filler)

### DISCUSSION THREAD

**Discussion by KukushkaSex** [Sokol-Games-GmbH/PixelCombats.PublicApi#78] [Ideas]
Ставит ассет по его имени из Кеша, или можно сделать сервер для этих ассетов...

**Comment by forwork10** [comment]
не имеет смысла

**Comment by kkohno** [comment]
идея крутая, это мы хотим сделать давно. вообще идея была в том, чтобы на гитхаб заливались ассеты и их спавнили потом режимы
